src/tests/ftest/soak/soak_faults.yaml

```diff
@@ -34,24 +34,26 @@ server_config:
         bdev_class: nvme
         bdev_list: ["0000:81:00.0","0000:da:00.0"]
 # pool_params - attributes of the pools to create; Currently only create one
-pool_ior:
+pool_jobs:
     mode: 146
     name: daos_server
-    scm_size: 80000000000
-    nvme_size: 200000000000
+    scm_size: 40GB
+    nvme_size: 100GB
     svcn: 1
     control_method: dmg
 pool_reserved:
     mode: 511
     name: daos_server
-    scm_size: 3000000000
-    nvme_size: 50000000000
+    scm_size: 3GB
+    nvme_size: 50GB
     control_method: dmg
 container:
     type: POSIX
     properties: cksum:crc16,cksum_size:16384,srv_cksum:on
+    oclass: RP_2G1
```

Comment:
Specific reason why only using 1 target and not all available?

Reply:
Should the oclass be RP_2GX?

Reply:
Yeah.